packages/run-protocol/src/vpool-xyk-amm/multipoolMarketMaker.js

```diff
@@ -291,10 +291,38 @@ const start = async (zcf, privateArgs) => {
     }),
   );
 
-  /** @type {GovernedCreatorFacet<*>} */
-  const creatorFacet = makeGovernorFacet(
+  // TODO: pass this into makeAddPoolInvitation
```

Comment:
I think this should be done before merge to DRY out the two addPools. They've already diverged in that [the other one updates metrics](https://github.com/Agoric/agoric-sdk/blob/0f4e0b53ffea8ea608f200752d8d342c0a2ce332/packages/run-protocol/src/vpool-xyk-amm/addPool.js#L97). cc @Chris-Hibbert